src/assets/compendiums/runes.db/runes.yaml

```diff
@@ -0,0 +1,618 @@
+name: Eternal Battle (condition)
+type: rune
+data:
+  rqid: rune-eternal-battle-condition
```

Comment:
These rqids need "i." in front of them.  Again, this PR isn't really about the content so I'm ok with leaving it.  We definitely need to discuss the rqid naming schema.